frontend/src/widgets/ResourceList.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withWidgetData from './withWidgetData';
+import TableWidget from './TableWidget';
+
+const renderResourceList = (data) => {
+  if (data && Array.isArray(data) && data.length > 0) {
+    const limitTDWith = { 'word-wrap': 'break-word', 'max-width': '500px' };
```

Comment:
I haven't looked at this in the browser yet so maybe it just works as is but React expects camel-cased versions of style attributes in the style object 
```{ wordWrap: 'break-word', maxWidth: '500px' }```

Reply:
```js
const renderResourceList = (data) => {
  if (data && Array.isArray(data) && data.length > 0) {
    return data.map((resource) => (
      <tr key={`resource_list_row_${resource.url}`}>
        <td className="maxw-mobile-lg" style={{ wordWrap: 'break-word' }}>
          <a href={resource.url}>{resource.name}</a>
        </td>
        <td>
          {resource.reportCount}
        </td>
        <td>
          {resource.participantCount}
        </td>
        <td>
          {resource.recipientCount}
        </td>
      </tr>
    ));
  }
  return null;
};
```